### PR TITLE
indents(vue): fix indents on new line after <template>

### PR DESCRIPTION
**Pull request by lucario387**
Same bug as #4076, #4086

### CODE REVIEW

**Review by amaanq — APPROVED** [review]

**Comment by amaanq** [comment]
Looks fine to me but I'm just unfamiliar with vue

File: queries/vue/indents.scm [hunk 1]
```diff
@@ -3,3 +3,5 @@
 [
   (template_element)
 ] @indent
+
+(template_element (end_tag "</" @branch ">" @indent_end))
```
[the remark this answers is not in the document]
Comment (lucario387) [reply queries/vue/indents.scm:7]: Yes, if not <template> is not dedented properly

[the remark this answers is not in the document]
Comment (amaanq) [reply queries/vue/indents.scm:7]: This still reads funny to me, what if you put `@branch` after the `>`. Does that work too?

[the remark this answers is not in the document]
Comment (lucario387) [reply queries/vue/indents.scm:7]: I don't know how to describe but the issue? starts from here 
https://github.com/nvim-treesitter/nvim-treesitter/blob/master/lua/nvim-treesitter/indent.lua#L115

An `indent_end` captured node cannot be branch, and a branch captured needs to be at the start or end of a line to have effect. Hence capturing `@branch` for `</` in `</template>`, or in other words, closing of the template tag

[the remark this answers is not in the document]
Comment (amaanq) [reply queries/vue/indents.scm:7]: ahhh I didn't know that...thanks

**Pull request merged**